src/run.js

```diff
@@ -218,6 +218,12 @@ const processPage = ({
           redirectResponses[responseUrl] = redirectsTo;
         } else if (resourceType === 'stylesheet') {
           response.text().then(text => {
+            // Semicolon sequences can crash CSSO, so we remove them.
+            // https://github.com/peterbe/minimalcss/issues/243
+            // https://github.com/css/csso/issues/378
+            while (/;\s*;/.test(text)) {
```

Comment:
we have file `src/utils.js` you can place this function there with explanations in jsdoc